packages/@snowpack/app-scripts-preact/babel.config.json

```diff
@@ -14,10 +14,5 @@
       }
     ]
   ],
-  "plugins": ["@babel/plugin-syntax-import-meta"],
-  "env": {
-    "development": {
-      "plugins": ["react-refresh/babel"]
-    }
-  }
```

Comment:
I don’t remember why react-refresh was being loaded for Preact when we have a prefresh plugin, but this was breaking.